### PR TITLE
Plat 606 keycloak openhim mapping refactor

### PR DESCRIPTION
**Pull request by nour-borgi**

### CODE REVIEW

File: .env.cluster [hunk 1]
```diff
@@ -27,10 +27,10 @@ REPMGR_PARTNER_NODES=santempi-psql-1,santempi-psql-2,santempi-psql-3
 
 # Reverse Proxy - Nginx
 REVERSE_PROXY_INSTANCES=3
+INSECURE=false
 DOMAIN_NAME=domain
-SUBDOMAINS=openhimcomms.domain,openhimcore.domain,openhimconsole.domain,kibana.domain,reports.domain,santewww.domain,santempi.domain,superset.domain,keycloak.domain,grafana.domain
+SUBDOMAINS=openhimcomms.domain,openhimcore.domain,openhimconsole.domain,kibana.domain,reports.domain,santewww.domain,santempi.domain,superset.domain,grafana.domain
```
Comment (marrouchi) [inline .env.cluster:32]: I wonder why did you remove keycloak.domain ?

File: identity-access-manager-keycloak/package-metadata.json [hunk 2]
```diff
@@ -6,7 +6,6 @@
   "version": "0.0.1",
   "dependencies": [],
   "environmentVariables": {
```
Comment (marrouchi) [inline identity-access-manager-keycloak/package-metadata.json:8]: I guess STATEFUL_NODE should be replaced by CLUSTER_MODE ?

File: monitoring/docker-compose.yml [hunk 3]
```diff
@@ -6,32 +6,13 @@ services:
...
-      GF_AUTH_GENERIC_OAUTH_EMAIL_ATTRIBUTE_PATH: "email"
-      GF_AUTH_GENERIC_OAUTH_LOGIN_ATTRIBUTE_PATH: "username"
-      GF_AUTH_GENERIC_OAUTH_NAME_ATTRIBUTE_PATH: "full_name"
-      GF_AUTH_GENERIC_OAUTH_AUTH_URL: "${KC_FRONTEND_URL}/realms/${KC_REALM_NAME}/protocol/openid-connect/auth"
-      GF_AUTH_GENERIC_OAUTH_TOKEN_URL: "${KC_API_URL}/realms/${KC_REALM_NAME}/protocol/openid-connect/token"
-      GF_AUTH_GENERIC_OAUTH_API_URL: "${KC_API_URL}/realms/${KC_REALM_NAME}/protocol/openid-connect/userinfo"
-      GF_AUTH_GENERIC_OAUTH_ROLE_ATTRIBUTE_PATH: "contains(roles[*], 'admin') && 'Admin' || contains(roles[*], 'editor') && 'Editor' || 'Viewer'"
-      GF_SERVER_DOMAIN: ${GF_SERVER_DOMAIN}
-      GF_SERVER_ROOT_URL: ${GF_SERVER_ROOT_URL}
+      - GF_SECURITY_ADMIN_USER=${GF_SECURITY_ADMIN_USER}
```
Comment (marrouchi) [inline monitoring/docker-compose.yml:9]: I wonder why you did remove some of the env vars, we need them for keycloak.

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: .env.cluster (hunk 1 above)
Comment (nour-borgi) [reply .env.cluster:32]: ops sorry merge request didn't notice it

File: monitoring/docker-compose.yml (hunk 3 above)
Comment (nour-borgi) [reply monitoring/docker-compose.yml:9]: same for keycloak a merge I didn't notice, fix it also

File: identity-access-manager-keycloak/package-metadata.json (hunk 2 above)
Comment (nour-borgi) [reply identity-access-manager-keycloak/package-metadata.json:8]: We agreed on removing this variable from package-metadata.json because it is not really an environment variable

**Review by bradsawadye — APPROVED** [review]
looks good. Please fix the merge conflicts
